networkx/algorithms/approximation/tests/test_clique.py

```diff
@@ -22,21 +22,21 @@
 
 
 def is_independent_set(G, nodes):
-    """Returns ``True`` if and only if ``nodes`` is a clique in ``G``.
+    """Returns True if and only if `nodes` is a clique in `G`.
```

Comment:
`True` (with a capital T) is a Boolean literal in Python, I would expect literals to be rendered as code. Do you prefer it without that styling?